index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap" rel="stylesheet">
+  <title>First mobile</title>
+</head>
+<body>
+  <header>
+    <div class="toolbar">
+      <a href="https://www.google.com">My Logo</a>
+      <button>
+        <img src="img/Union.png" alt="wite bars">
+      </button> 
+    </div>
+  </header>
+    <main>
+      <section class="head">
+        <div class="container">
+          <h1>Hey There.<br>I’m Ricardo.</h1>
+          <h2>I'm a Software Developer</h2>
+          <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hesitate to contact me.</p>
+          <img src="img/Social Media.png" alt="social"/>
```

Comment:
- [ ] Great job with the social icons. unfortunately its not an image but they a different social icons which are aimed at linking the viewer to your different social profiles. please the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).

* Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.